src/components/app/app-authenticated/app-authenticated.module.css

```diff
@@ -3,6 +3,11 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+body {
```

Comment:
why body? I don't think the component includes a body tag. The body tag is higher up in the react DOM tree.